### ISSUE TITLE
text color contrast does not meet minimum requirements on edit file page

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6810]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `Who should be able to view or download this content?` text on the `Permissions` tab of the `Edit File` page does not meet minimum WCAG AA color contrast requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
As an admin, go to https://dev.nurax.samvera.org/concern/file_sets/p2676v71k/edit?locale=en#permissions_display and run the Site Improve browser extension
![edit_file_color_contrast](https://github.com/samvera/hyrax/assets/32644254/07b9c564-f210-4b87-976a-ac2612b833df)
